### PR TITLE
Update ETC_en.tsv

### PR DESCRIPTION
**Pull request by morridinn**
Translated line 97, 118, 121, 124, 136, 137, 138, 142, 173, 175, 176, 177, 178, 181, 204, 220, 223, 229

### CODE REVIEW

File: ETC_en.tsv [hunk 1]
```diff
@@ -201,7 +201,7 @@ ETC_20150312_001699	You have to stop the Rexipher!!!
 ETC_20150312_001700	Aarrgh!!!!
 ETC_20150312_001701	Help me !!!
 ETC_20150312_001702	If you disturb me again..
-ETC_20150312_001703	
+ETC_20150312_001703	This woman will dissapear from the world.
```
Comment (laurencee) [inline ETC_en.tsv:204]: Spelling, should be "disappear"

File: ETC_en.tsv [hunk 2]
```diff
@@ -170,15 +170,15 @@ ETC_20150312_001668	Artifact of royal tomb
...
+ETC_20150312_001672	The first seal of Sanctuary
 ETC_20150312_001673	A secondary seal device was summoned when you touched the seal!{nl}Destroy the secondary seal device!
-ETC_20150312_001674	
-ETC_20150312_001675	
-ETC_20150312_001676	
-ETC_20150312_001677	
+ETC_20150312_001674	Second seal of Sanctuary
+ETC_20150312_001675	Secondary sealing device
+ETC_20150312_001676	Third seal of Sanctuary
+ETC_20150312_001677	Ancient Pilliars
```
Comment (laurencee) [inline ETC_en.tsv:178]: I presume this is meant to be "Pillars" ?

File: ETC_en.tsv [hunk 3]
```diff
@@ -115,13 +115,13 @@ ETC_20150312_001613	Where did the Goddess go..
 ETC_20150312_001614	Give us peace and salvation..
 ETC_20150312_001615	Listen to our prayers..
 ETC_20150312_001616	Peace and happiness.
-ETC_20150312_001617	
+ETC_20150312_001617	There! The spider is coming!
 ETC_20150312_001618	Lylia suddenly turned into a carnivore!
 ETC_20150312_001619	Lylia that changed to carnivore
-ETC_20150312_001620	
+ETC_20150312_001620	Krylian Pilgrems*
```
Comment (laurencee) [inline ETC_en.tsv:121]: Spelling, "Pilgrims"

**Comment by imcgames** [comment]
:+1:

**Pull request merged**